website/guides/core-workflow.html.md

```diff
@@ -0,0 +1,335 @@
+
+In addition to reviewing the plan for the proper expression of its author's
+intent, the team can also make an evaluation whether they want this change to
+happen now. For example, if a team notices that a certain change could result
+in service disruption, they may decide to delay merging its pull request until
+they can schedule a maintenance window.
+
+### Apply
+
+Once a pull request has been approved and merged, it's important for the team
+to review the final concrete plan that's run against the shared team branch and
+the latest version of the state file.
+
+This plan has the potential to be different than the one reviewed on the pull
+request due to issues like merge order or recent infrastructural changes. For
+example, if a manual change was made to your infrastructure since the plan was
+reviewed, the plan might be different when you merge.
+
+It is at this point that the team asks questions about the potential
+implications of applying the change. Do we expect any service disruption from
+this change? Is there any part of this change that is high risk? Is there
+anything in our system that we should be watching as we apply this? Is there
+anyone we need to notify that this change is happening?
+
+Depending on the change, sometimes team members will want to watch the apply
+output as it is happening. For teams that are running Terraform locally, this
+may involve a screen share with the team. For teams running Terraform in CI,
+this may involve gathering around the build log.
+
+Just like the workflow for individuals, the core workflow for teams is a loop
+that plays out for each change. For some teams this loop happens a few times a
+week, for others, many times a day.
+
+## The Core Workflow Enhanced by Terraform Enterprise
+
+While the above described workflows enable the safe, predictable, and
+reproducible creating or changing of infrastructure, there are multiple
+collaboration points that can be streamlined, especially as teams and
+organizations scale.  We designed Terraform Enterprise to support and enhance
+the core Terraform workflow for anyone collaborating on infrastructure, from
+small teams to large organizations. Let's look at how Terraform Enterprise makes
+for a better experience at each step.
+
+### Write
+
+Terraform Enterprise provides a centralized and secure location for storing
+input variables and state while also bringing back a tight feedback loop for
+speculative plans for config authors. Terraform configuration interacts with
+Terraform Enterprise via the ["remote" backend](/docs/backends/types/remote.html).
+
+```
+terraform {
+  backend "remote" {
+    organization = "my-org"
+    workspaces {
+      prefix = "my-app-"
+    }
+  }
+}
+```
+
+Once the backend is wired up, a Terraform Enterprise API key is all that's
+needed by team members to be able to edit config and run speculative plans
+against the latest version of the state file using all the remotely stored
+input variables.
+
+```sh
```

Comment:
I wouldn't `sh` this, because it's a lot more free-form Terraform output than it is shell, and it's likely to randomly highlight bits of punctuation and apparent keywords in a distracting way.

Reply:
Fair play, will fix.